src/runtime/agas/server/primary_namespace_server.cpp

```diff
@@ -56,11 +56,8 @@ response primary_namespace::service(
     {
         case primary_ns_route:
             {
-                update_time_on_exit update(
-                    counter_data_.route_.time_
-                );
-                counter_data_.increment_route_count();
-                return route(req, ec);
+                HPX_ASSERT(false);
+                return response();
```

Comment:
Have you removed the corresponding performance counters as well? Or have you moved their data collection to the new route action?